arm-datalake-store/account/2016-11-01/swagger/account.json

```diff
@@ -0,0 +1,1321 @@
+            "modelAsString": false
+          }
+        },
+        "principalId": {
+          "type": "string",
+          "format": "uuid",
+          "readOnly": true,
+          "description": "The principal identifier associated with the encryption."
+        },
+        "tenantId": {
+          "type": "string",
+          "format": "uuid",
+          "readOnly": true,
+          "description": "The tenant identifier associated with the encryption."
+        }
+      },
+      "description": "The encryption identity properties."
+    },
+    "EncryptionConfig": {
+      "required": [
+        "type"
+      ],
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "The type of encryption configuration being used. Currently the only supported types are 'UserManaged' and 'ServiceManaged'.",
+          "enum": [
+            "UserManaged",
+            "ServiceManaged"
+          ],
+          "x-ms-enum": {
+            "name": "EncryptionConfigType",
+            "modelAsString": false
+          }
+        },
+        "keyVaultMetaInfo": {
+          "$ref": "#/definitions/KeyVaultMetaInfo",
+          "description": "The Key vault information for connecting to user managed encryption keys."
+        }
+      },
+      "description": "The encryption configuration for the account."
+    },
+    "KeyVaultMetaInfo": {
+      "required": [
+        "keyVaultResourceId",
+        "encryptionKeyName",
+        "encryptionKeyVersion"
+      ],
+      "properties": {
+        "keyVaultResourceId": {
+          "type": "string",
+          "description": "The resource identifier for the user managed Key Vault being used to encrypt."
+        },
+        "encryptionKeyName": {
+          "type": "string",
+          "description": "The name of the user managed encryption key."
+        },
+        "encryptionKeyVersion": {
+          "type": "string",
+          "description": "The version of the user managed encryption key."
+        }
+      },
+      "description": "Metadata information used by account encryption."
+    },
+    "DataLakeStoreAccountProperties": {
+      "properties": {
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "description": "the status of the Data Lake Store account while being provisioned.",
+          "enum": [
+            "Failed",
+            "Creating",
+            "Running",
+            "Succeeded",
+            "Patching",
+            "Suspending",
+            "Resuming",
+            "Deleting",
+            "Deleted"
+          ],
+          "x-ms-enum": {
+            "name": "DataLakeStoreAccountStatus",
+            "modelAsString": false
+          }
+        },
+        "state": {
+          "readOnly": true,
+          "type": "string",
+          "description": "the status of the Data Lake Store account after provisioning has completed.",
+          "enum": [
+            "active",
```

Comment:
This is so weird. "encryptionState" and "provisioningState" has enum values in PascalCase whereas state has all the values in lowercase. I hope this is the actual wire representation. It would be nice if a consistent approach is taken w.r.t enum values for the enitre RP (Azure wide would be even better).